docs/node/activities.md

```diff
@@ -0,0 +1,47 @@
+# Activities
+
+In Temporal, Activities are typically used to represent side effects, like making an HTTP request.
+Unlike [Workflows](/docs/node/logging#logging-from-workflows-and-activities), activities run in the standard Node.js environment, not an [isolate](https://www.npmjs.com/package/isolated-vm).
+So any code that needs to talk to the outside world, including importing modules, needs to be in an Activity.
```

Comment:
Not true, you can import any module in WFs as long as it doesn't use the node or dom libs.